android/app/src/main/java/com/arthenica/mobileffmpeg/AbiDetect.java

```diff
@@ -35,7 +36,16 @@
 
         /* LOAD NOT-LOADED LIBRARIES ON API < 21 */
         if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
-            System.loadLibrary("cpufeatures");
+            try {
+                System.loadLibrary("c++_shared");
```

Comment:
This was introduced to allow loading **libavcodec**. There was some unresolved symbol before (in case of LTS build), but I don't remember exactly how it happened. Anyways, this is closely related to https://github.com/tanersener/mobile-ffmpeg/pull/375#discussion_r427548560.